src/app/icd/client/RefreshKeySender.cpp

```diff
@@ -64,6 +64,9 @@ CHIP_ERROR RefreshKeySender::RegisterClientWithNewKey(Messaging::ExchangeManager
         }
 
         mpCheckInDelegate->OnCheckInComplete(mICDClientInfo);
+#if CHIP_CONFIG_ENABLE_READ_CLIENT
+        InteractionModelEngine::GetInstance()->OnActiveModeNotification(mICDClientInfo.peer_node);
```

Comment:
Again, should this be injected somehow?  Followup fine for that.

Reply:
updated, thanks